### PR TITLE
Use ynh_handle_getopts_args

### PR DESCRIPTION
**Pull request by maniackcrudelis**
Use ynh_handle_getopts_args with all our experimental helpers.
If one of your helpers is modified by this PR, please have a look and express yourself if you have anything to say about it. Even if it's to say that you don't want to use getopts for your helper.

### CODE REVIEW

File: ynh_read_manifest/ynh_read_manifest_2 [hunk 1]
```diff
@@ -23,7 +30,7 @@ ynh_app_upstream_version () {
     if [ ! -e "$manifest_path" ]; then
         manifest_path="../settings/manifest.json"	# Into the restore script, the manifest is not at the same place
     fi
-    version_key=$(ynh_read_manifest "$manifest_path" "version")
+    version_key=$(ynh_read_manifest --manifest "$manifest_path" --key "version")
```
Comment (JimboJoe) [inline ynh_read_manifest/ynh_read_manifest_2]: Wrong syntax... :wink:

File: ynh_read_manifest/ynh_read_manifest_2 [hunk 2]
```diff
@@ -39,6 +46,6 @@ ynh_app_package_version () {
...
```
Comment (JimboJoe) [inline ynh_read_manifest/ynh_read_manifest_2]: Wrong syntax... :wink:

File: ynh_read_manifest/ynh_read_manifest_2 (hunk 1 above)
Comment (maniackcrudelis) [reply ynh_read_manifest/ynh_read_manifest_2]: :D
Not me ! :p

File: ynh_add_swap/ynh_add_swap [hunk 3]
```diff
@@ -1,8 +1,16 @@
 #!/bin/bash
 
+# Need also the helper https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_handle_getopts_args/ynh_handle_getopts_args
+
 # Argument $1 is the size of the swap in MiB
```
Comment (Josue-T) [inline ynh_add_swap/ynh_add_swap:5]: Did you forgot to replace `$1` ?

Comment (maniackcrudelis) [reply ynh_add_swap/ynh_add_swap:5]: Not really, it's more a matter of structured comment missing here. As you can see.

**Comment by Josue-T** [comment]
Can we go forward with this PR ?

**Comment by maniackcrudelis** [comment]
I was waiting for each of you to check his own helpers before merging it.
We can say that in 3 days, we will merge this PR is no one express any oppositions...

**Comment by JimboJoe** [comment]
I didn't get the time to adapt and test some apps, but don't oppose to merging it!

**Comment by Josue-T** [comment]
For me it's ok if we merge this in 3 days.

**Comment by maniackcrudelis** [comment]
Let's be crazy, let's merge it NOW !!!

**Comment by frju365** [comment]
I agreed too. It was totally ok to merge... (I'm too late)

**Pull request merged**